src/classes/Commands.ts

```diff
@@ -2247,7 +2247,8 @@ export = class Commands {
 
             reply += `\n- Offer #${offer.id} from ${
                 offer.data.partner
-            } (reason: ${offer.data.action.meta.uniqueReasons.join(', ')})`;
+            } (reason: ${offer.data.action.meta.uniqueReasons.join(', ')})
+            ⚠️ Send "!trade ${offer.id}" for more details.\n `;
```

Comment:
please separate this, and why you put `\n` at the end of it?
```
            reply += `\n- Offer #${offer.id} from ${
                offer.data.partner
            } (reason: ${offer.data.action.meta.uniqueReasons.join(', ')})` +
                '⚠️ Send "!trade ${offer.id}" for more details.';
```

Reply:
With `\n` at the end
![image](https://user-images.githubusercontent.com/10933535/97084465-a5eae800-15ed-11eb-904a-2c0181cfa346.png)

Without `\n` at the end
![image](https://user-images.githubusercontent.com/10933535/97084408-28bf7300-15ed-11eb-9195-595c46af4d92.png)
I like it more with `\n` at the end

Reply:
oh I see.. alright you can keep it 👍